.gitignore

```diff
@@ -6,3 +6,6 @@ package-lock.json
 
 coverage/
 public/
+client/dist
+src/dist
+dist/
```

Comment:
gitignore should probably use leading `/` because `dist/` right now will cover any subfolder.